app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/bta/BtaEntryPointController.scala

```diff
@@ -34,4 +34,9 @@ class BtaEntryPointController @Inject() (
       Redirect(returnRoutes.StartDateReturnController.displayPage())
     }
 
+  def submittedReturns(): Action[AnyContent] =
+    authenticate { implicit request =>
+      Redirect(returnRoutes.SubmittedReturnsController.displayPage())
```

Comment:
Can the route not just point to this controller? save on the double calls to auth?

Reply:
or can we just redirect without auth

Reply:
Cant remeber how it works but could probs use the Default controller to do this

Reply:
https://www.playframework.com/documentation/2.8.x/api/scala/controllers/Default.html

Reply:
`DefaultController` looks interesting. I wonder whether you can use the reverse routing. I'll take a quick look...

Reply:
Couldn't see an obvious way of doing this.